# Cappucino

## Introductie
Heb je al eens bedacht dat het snel verwarmen van melk voor een cappucino eigenlijk heel snel gaat zonder dat de melk verdunt wordt?
De melk wordt ook niet echt plaatselijk verhit door een warmte element.
In plaats daarvan wordt er stoom door de melk geblazen.
De stoom condenseert in de melk en geeft daarbij zijn latente warmte af.
Hierdoor warmt de melk snel op zonder dat er (significant veel) water aan toegevoegd wordt.

```{warning}
Stoom is heel heet en kan voor vervelende brandplekken zorgen.
Wees dus voorzichtig bij het uitvoeren van dit practicum.

Daarnaast, nadat de kookplaat uitgaat is er een onderdruk in de kolf.
Hang dan ook de tube uit het water wanneer je de kookplaat uitzet, anders kan de kolf imploderen.
```

In dit practicum gaan we de verdampingswarmte van water bepalen door middel van een zelfgebouwde cappucino machine.
Het principe is als volgt: We gebruiken de opstelling weergegeven in {numref}`Figuur {number} <fig_cappucino>` waarbij we water verwarmen in een kolf met behulp van een kookplaat.
Door de warmte gaat het water koken en ontstaat er stoom.
De stoom stroomt via een tube naar een afgesloten maatbeker met water.
De stoom condenseert in het maatbeker en geeft daarbij zijn latente warmte af.
Door de temperatuurstijging van het water te meten, evenals de hoeveelheid gram water dat verdampt is, kunnen we de verdampingswarmte van water bepalen.

```{figure} figures/verdampingswarmte_setup.png
:width: 70%
:label: fig_cappucino

Een schematische weergave van de cappucino opstelling.
```

## Theorie

De latente verdampinswarmte van water bedraagt 2257 kJ/kg, dit is veel meer dan de specifieke warmtecapaciteit van water (4.18 kJ/kgK).
Wanneer we de waterdamp (g) door koud water leiden, gaan we er van uit dat het waterdamp condenseert en daarbij zijn latente warmte afgeeft aan het koude water.
Door te bepalen hoeveel gram water verdampt is en hoeveel de temperatuur van het koude water stijgt, kunnen we de latente warmte van verdamping bepalen:

$$ Q_{condensatie} = m_{damp} L = m_{water} c \Delta T $$

met $m_{damp} = \Delta m_{kolf}$.

## Methoden en materialen

### Materialen
- Warmteplaat
- Kolf met stop en tube
- Maatcilinder
- Thermometer
- Weegschaal
- Water


### Procedure

- Vul de kolf met ongeveer 100 mL water: bepaal precies de massa water ($m_{w_1}$).  
- Vul de maatcilinder met ongeveer 100 mL water: bepaal precies de massa water ($m_{bad}$).
- Bepaal de temperatuur van dit waterbad ($T_{bad_1}$).
- Zet de kolf op de warmteplaat en zet de warmteplaat aan - maximale stand 3, zorg ervoor dat de tube goed in het waterbad hangt.
- Wacht tot de temperatuur van het waterbad met ongeveer 20 K is gestegen.
- Stop de meting / zet de warmteplaat uit. Noteer meteen de temperatuur($T_{bad_2}$) en haal de tube uit het water! 

```{warning}
Pas op stoom is heet!
```

- Bepaal precies de massa water in de kolf ($m_{w_2}$).
- Verwerk je resultaten hieronder om de latente warmte van verdamping van water te bepalen.

## Resultaten

In [26]:
# Kolf: 
m_kolf_0 = 122.672*1e-3 # Massa van de kolf in kg.
m_kolf_b = 207.583*1e-3 # Massa van de kolf mét ~100 mL water aan het begin in kg.
m_kolf_e = 199.747*1e-3 # Massa van de kolf mét ~100 mL water aan het eind in kg.

# Maatcilinder: 
m_maat_0 = 109.460*1e-3 # Massa van de maatcilinder in kg.
m_maat_b = 207.581*1e-3 # Massa van de maatcilinder mét water aan het begin in kg.

# 20K verwarmen (dus 20 graden Celsius verwarmen, want de tijdsstappen zijn even groo)
delta_T = 20

m_w_1 = m_kolf_b - m_kolf_0      # Massa van het water in de kolf aan het begin in kg.
m_bad = m_maat_b - m_maat_0      # Massa van het water in de maatcilinder in kg.
T_bad_1 = 17.4                   # Begintemperatuur van het water in de maatcilinder in graden Celsius

m_w_2 = m_kolf_e - m_kolf_0      # Massa van het water in de kolf aan het eind in kg.
T_bad_2 = 37.4                   # Eindtemperatuur van het water in de maatcilinder in graden Celsius.

m_damp = m_w_1 - m_w_2
c = 4.18                         # Specifieke warmtecapaciteit van water in kJ/kgK

L_water = (m_bad * c * delta_T) / m_damp

print('De gevonden waarde voor de latente warmte van de verdamping van water is %.3f kJ.' % (L_water))

De gevonden waarde voor de latente warmte van de verdamping van water is 1046.824 kJ.


## Discussie en conclusie

Er druppelde ook water terug de kolf in, de temperatuur werd op 1 punt gemeten (niet geroerd) waardoor de temperatuur op dat punt niet de gemiddelde waarde van het water in de maatcilinder kan zijn geweest.

# Discussie

De experimenteel bepaalde latente verdampingswarmte van water bedraagt 1046.824 kJ/kg, wat aanzienlijk lager is dan de literatuurwaarde van 2257 kJ/kg. Deze afwijking kan worden verklaard door verschillende factoren die allemaal leiden tot een onderschatting van de temperatuurstijging van het water in de maatcilinder of een overschatting van de hoeveelheid warmte die daarin is afgegeven. 

Een belangrijke foutbron is dat de temperatuur van het water in de maatcilinder slechts op één punt is gemeten, zonder roeren. Doordat de stoom lokaal condenseert, ontstaan sterke temperatuurverschillen in de maatcilinder. Wanneer de thermometer zich in een relatief koud gebied bevindt, wordt de temperatuurstijging te klein gemeten, omdat de berekening van de latente warmte direct evenredig is met deze temperatuurstijging leidt dit tot een te lage waarde voor "L".

Daarnaast is niet alle stoom volledig in het water in de maatcilinder gecondenseerd. Een deel van de stoom kan al in de tube of tegen de wand van de maatcilinder zijn gecondenseerd of zelfs ontsnapt zijn zonder warmteoverdracht. In dat geval wordt de massa-afname van de kolf wel juist meegenomen, maar komt slechts een deel van de bijbehorende latente warmte in het water in de maatcilinder terecht. Hierdoor wordt een te kleine temperatuurstijging gemeten.

Ook warmteverlies aan de omgeving speelt een rol. De maatcilinder is niet geïsoleerd, waardoor een deel van de warmte die door condensatie vrijkomt direct verloren gaat aan de lucht en de wand van de cilinder. Dit verlaagt de gemeten temperatuurstijging verder en draagt dus bij aan de te lage experimentele waarde ten opzichte van de literatuurwaarde.

### Vervolgonderzoek

Om beter te begrijpen hoeveel warmte er precies verloren gaat aan de omgeving kan het experiment worden herhaald met en zonder isolatie rond de maatcilinder (bijvoorbeeld met een handdoek of piepschuim). Door de gemeten temperatuurstijgingen en de berekende waarden van "L" in beide situaties te vergelijken, kan worden vastgesteld hoeveel warmte het water in de maatcilinder aan zijn omgeving geeft. Dit kan laten zien in hoeverre isolatie de nauwkeurigheid van de metingen verbetert.

## Conclusie

In dit experiment is de latente verdampingswarmte van water bepaald met behulp van een zelfgebouwde opstelling waarbij stoom door het water in de maatcilinder werd geleid. De berekende waarde bedraagt 1046.824 kJ/kg wat lager ligt dan de literatuurwaarde van 2257 kJ/kg. Deze afwijking wordt voornamelijk veroorzaakt door een onderschatte temperatuurstijging van het water in de maatcilinder, doordat de temperatuur slechts op één punt werd gemeten en het water niet werd geroerd. Daarnaast heeft onvolledige condensatie van stoom in de maatcilinder en warmteverlies aan de omgeving geleid tot een lagere gemeten opwarming. Hoewel de gemeten waarde afwijkt van de literatuurwaarde, laat het experiment wel zien dat bij condensatie veel warmte vrijkomt en dat de latente verdampingswarmte van water van de juiste orde van grootte is.